_data/versions.json

```diff
@@ -1,12 +1,13 @@
 {
-  "current": "1.3",
+  "current": "2.1",
```

Comment:
This is the 1.3 branch. `current` should remain `1.3`.

Reply:
Done